### PR TITLE
streamlined setup

### PR DESCRIPTION
**Pull request by magician11**
Setup environment variables
Add requirements.txt file

### CODE REVIEW

**Review by hartnady — APPROVED** [review]

File: run.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import openai, time, json, transformers, re, datetime, sys, os
 from mailmerge import MailMerge #pip install docx-mailmerge
 from tqdm import tqdm, trange
+from dotenv import load_dotenv
```
Comment (hartnady) [inline run.py:4]: load_dotenv is not needed as the os module already used. You can instead use api_key = os.environ.get('OPENAI_KEY')

File: run.py [hunk 2]
```diff
@@ -11,8 +12,11 @@
 GPT_CALL_OUT_ON = True   #To save costs, set to False when debugging or developing
 PRINT_AS_YOU_GO = False
 
+# dotenv_path = join(dirname(__file__), '.env')
+load_dotenv()
```
Comment (hartnady) [inline run.py:15]: See other comment

File: run.py [hunk 3]
```diff
@@ -11,8 +12,11 @@
...
+
 # Set up your OpenAI API key
-openai.api_key = "YOUR-API-KEY-HERE"
+openai.api_key = os.environ.get("OPENAI_API_KEY")
```
Comment (hartnady) [inline run.py:17]: Yes!

**Review by hartnady — CHANGES_REQUESTED** [review]

File: run.py (hunk 1 above)
Comment (magician11) [reply run.py:4]: python_dotenv module is needed if you want to use .env files to standardise how environment variables are set across OS's.

File: run.py (hunk 3 above)
Comment (magician11) [reply run.py:17]: Then yes you can access .env variables like any other OS specific environment variable

File: run.py (hunk 1 above)
Comment (hartnady) [reply run.py:4]: Nice one!

**Pull request merged**